tests/integration_tests/with_db/deployment/test_scripts_deployer_cli.py

```diff
@@ -34,7 +34,8 @@ def test_scripts_deployer_cli(backend,
         assert not result.exception
         assert result.exit_code == 0
         assert DBQueries.check_all_scripts_deployed(
-            pyexasol_connection, schema_name)
+            pyexasol_connection, schema_name, expected_script_list_without_span)
```

Comment:
ah, this test we need to change at some point to a unit test using a mock of the scripts_deployer api. Could you create a ticket for that.

Reply:
https://github.com/exasol/transformers-extension/issues/263